src/upgraider/populate_doc_db.py

```diff
@@ -1,5 +1,5 @@
 
-from Model import get_embedding
+from promptCrafting import get_embedding
```

Comment:
thanks for spotting this. We can change for now but if `get_embedding` actually makes more sense to be part of Model in the future, you can refactor it later when you have a better idea of the code structure and what can be improved